CesiumGltfContent/test/TestIntersectRayGltf.cpp

```diff
@@ -0,0 +1,240 @@
+#include "CesiumGeometry/IntersectionTests.h"
+#include "CesiumGeometry/Ray.h"
+#include "CesiumGltfContent/GltfUtilities.h"
+#include "CesiumGltfReader/GltfReader.h"
+
+#include <CesiumUtility/Math.h>
+
+#include <catch2/catch.hpp>
+#include <glm/glm.hpp>
+
+#include <filesystem>
+#include <fstream>
+
+using namespace CesiumUtility;
+using namespace CesiumGltf;
+using namespace CesiumGeometry;
+using namespace CesiumGltfReader;
+using namespace CesiumGltfContent;
+
+namespace {
+std::vector<std::byte> readFile(const std::filesystem::path& fileName) {
+  std::ifstream file(fileName, std::ios::binary | std::ios::ate);
+  REQUIRE(file);
+
+  std::streamsize size = file.tellg();
+  file.seekg(0, std::ios::beg);
+
+  std::vector<std::byte> buffer(static_cast<size_t>(size));
+  file.read(reinterpret_cast<char*>(buffer.data()), size);
+
+  return buffer;
+}
+} // namespace
```

Comment:
Use the implementation of this already in `readFile.h` / `readFile.cpp` instead of copying it here.

Reply:
Done. Also fixed up some other duplicate occurrences